app/src/main/java/com/github/factotum_sdp/factotum/placeholder/Contact.kt

```diff
@@ -0,0 +1,14 @@
+package com.github.factotum_sdp.factotum.placeholder
+
+data class Contact(
+    val id: String,
+    val role: String,
+    val name: String,
+    val surname: String,
+    val profile_pic_id: Int,
+    val address: String,
+    val phone: String,
+    val details: String? = null)
+{
+    constructor() : this("", "", "", "",0, "", "", null)
+}
```

Comment:
No need to specify constructor explicitly.
You can simply do it like so :
`val id: String = ""`

Reply:
Thanks, I didn't know that.